lorekeeper.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'oj', '~> 2.18'
+  spec.add_dependency 'oj', '~> 3.3'
```

Comment:
should be more like `>= 3.3.1` because `~> 3.3` means that the people could use 3.3.0, which doesn't include the fix.